src/services/leafletEvents.js

```diff
@@ -1,6 +1,10 @@
 angular.module("leaflet-directive").factory('leafletEvents',
     function (leafletMapEvents, leafletMarkerEvents, leafletPathEvents) {
-        var retObj = {};
-        angular.extend(retObj,leafletMapEvents, leafletMarkerEvents, leafletPathEvents);
-        return retObj;
+      //NOTE THIS SHOULD BE DEPRECATED infavor of getting a specific events helper
```

Comment:
This file really should be deleted as it is a connivence that causes more havoc on the development side.